### PR TITLE
Fix escaped quote string syntax highlighting

### PR DESCRIPTION
**Pull request by xykj61**
tl;dr: Fixes this:

language-hoon

Fixes an issue where escaped single or double quotes aren't parsed correctly and
correctly-closed Hoon string quotes continue to "stringify" the rest of
a .hoon file in Atom until the next closing respective single or double quote.

Also, @yonilevy, since a number of Urbit programmers are using this
package, would it be cool if you let us maintain this repo and APM
management in an official github/urbit repo? We think it'd be helpful to the community
to have it beside our Vim, Emacs, and Sublime editor tools for Hoon (which also
all need serious improvement), and having everything in one place is a good place
to start encouraging contribution. Also, it helps the Urbit community to have our 
official logo on APM when you search for the package in Atom.

Feel free to reach out to me directly via email or Twitter if this is
something you're cool with! We'll still credit you of course. :)
(Serious thank-you for creating this in the first place!)

### CODE REVIEW

**Comment by Fang-** [comment]
Keaton didn't mention, but I imagine his request also goes for the hoon-assist repo.

**Comment by xykj61** [comment]
Oh yeah! That'd be great too.

**Comment by yonilevy** [comment]
Hey guys!
I love that you're using this stuff :)

As you might be able to tell I've abandoned this project (due to a serious lack of spare time), so would actually be great if *anyone* else maintains it, let alone the official urbit account...

I'll contact you via email to coordinate.
